content/200-orm/200-prisma-client/500-deployment/001-deploy-prisma.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: 'Deploy Prisma'
-metaTitle: 'Deploying Prisma-based projects'
+title: 'Deploy Prisma Client'
+metaTitle: 'Deploying Prisma Client-based projects'
```

Comment:
I think you creted this page @jharrell, do you also feed that these are more "Prisma ORM" and not just "Prisma Client" speific although the meta description uses that term?

Reply:
Correct. This should be Prisma ORM and not Client 
```suggestion
title: 'Deploy Prisma ORM'
metaTitle: 'Deploying Prisma ORM-based projects'
```